webots_ros2_ur_e_description/package.xml

```diff
@@ -6,7 +6,7 @@
   <description>Universal Robot description for Webots.</description>
 
   <maintainer email="support@cyberbotics.com">Cyberbotics</maintainer>
-  <license>Apache License 2.0</license>
+  <license>BSD</license>
```

Comment:
Should this have been an addition instead of a replacement?

It would be fine for the other files in this repository to be licensed Apache v2.

If you wish to switch the package to Apache, then the note in the `README.md` about the exception for the `.urdf` files would no longer be needed.

Reply:
It is fine for us to publish this whole package as BSD (this concerns only this package, the other packages on this repo remain Apache-2) so that the license remains the same as the one of the package it is based on.